katello/hooks/pre_commit/13-hiera.rb

```diff
@@ -1,17 +1,48 @@
-TUNING_SIZES = ['default'] + ['medium', 'large', 'extra-large', 'extra-extra-large']
+# A mapping of tuning profile to the required CPU cores and memory in GB
+TUNING_SIZES = {
+  'default' => {cpu_cores: 1, memory: 8},
```

Comment:
didn't you say you wanted be ruby 2.0 compatible? :)

Reply:
Turns out that's Ruby 1.9 that introduced this syntax. I tested it on EL7 and it worked as expected so I'm good with it :)

Reply:
:D